src/main/java/org/sonatype/nexus/repository/puppet/internal/PuppetRecipeSupport.groovy

```diff
@@ -96,13 +105,58 @@ abstract class PuppetRecipeSupport
   @Inject
   Provider<PurgeUnusedFacet> purgeUnusedFacet
 
-  @Inject
-  Provider<NegativeCacheFacet> negativeCacheFacet
-
-  @Inject
-  NegativeCacheHandler negativeCacheHandler
-
   protected PuppetRecipeSupport(final Type type, final Format format) {
     super(type, format)
   }
+
+  /**
+   * Matcher for module releases.
+   */
+  static Matcher moduleReleasesSearchByNameMatcher() {
```

Comment:
Mostly it's to set a different AssetKind, since the responses are genuinely different. Figured that might be helpful in the future if we have to do something specific with the responses. I realize you could use the other way of applying an AssetKind and using a common matcher, but I've actually preferred the readability that you get from doing it this way (even if there is some duplication of code).